apps/browser/src/vault/popup/services/vault-popup-items.service.ts

```diff
@@ -133,14 +135,24 @@ export class VaultPopupItemsService {
       shareReplay({ refCount: true, bufferSize: 1 }),
     );
 
+  private userHasArchive$ = this.activeUserId$.pipe(
+    switchMap((userId) => {
+      return this.cipherArchiveService.userCanArchive$(userId);
+    }),
+  );
```

Comment:
nit: (non-blocking) There is a subtle switch between the variable names here that could be confusing: `userHasArchive` vs `userCanArchive`. This could come into play in the scenario where a user loses their premium status but still has an archive. 

```suggestion
  private userCanArchive$ = this.activeUserId$.pipe(
    switchMap((userId) => {
      return this.cipherArchiveService.userCanArchive$(userId);
    }),
  );
```